src/architecture/storage.md

```diff
@@ -200,63 +200,54 @@ Exonum uses third-party database engines to persist blockchain state
 locally. To use the particular database, a minimal [`Database`][database]
 interface should be implemented for it:
 
-- Get value by key
-- Put new value at the key (insert or update the saved one)
-- Delete key-value pair by key
+- Get a value by a [column family][col-family] name and a key
+- Put a new value at the specified column family / key (insert or update
+  the saved one)
+- Delete a key-value pair by column family name / key
 
 All the tables functionality is reduced to these atomic call types.
 
-As of Exonum 0.1, [LevelDB][level-db] is used as the database engine and
-as of Exonum 0.2 [RocksDB][rocks-db] support is realized.
+As of Exonum 0.3, the main database engine is [RocksDB][rocks-db].
+In versions 0.1 and 0.2, [LevelDB][level-db] was supported as well, but
+since 0.3, its support has been dropped.
 
-All the values from different tables are stored in one big key-value
-table at the low-level storage, wherein the keys are represented as
+Values from different tables are stored in column families in the low-level storage,
+wherein the keys are represented as
 a byte sequence, and values are serialized according to Exonum binary
-serialization
-format. Keys of the wrapped `BaseIndex` of a specific table
+serialization format. A single column family may store data for
+more than one table (see table groups below).
+Keys of the wrapped `BaseIndex` of a specific table
 are mapped to the low-level storage keys
 in a deterministic manner using [table identifiers](#table-identifiers).
 
 ### Table Identifiers
 
-Every table is uniquely identified by the compound prefix, which is used
-to map table keys into keys of the underlying low-level storage. The
-keys are prepended with this prefix which is unique to each table, and thus
-allows to distinguish values from different tables.
+Every table is uniquely identified by a compound identifier, which is used
+to map table keys into a column family and its keys in the underlying
+low-level storage. A table identifier consists of 2 parts:
 
-The table prefix consists of [the service ID](services.md#service-identifiers)
-and an internal identifier inside the
-service.
-All tables created with the same prefix will be the views of the same data.
+- String name, which is mapped 1-to-1 to a column family.
+  The name may contain uppercase and lowercase Latin letters, digits,
+  and underscores `_`.
```

Comment:
This is where it gets a bit unclear. [The source](https://github.com/exonum/exonum/blob/v0.3/exonum/src/storage/base_index.rs#L249) shows that periods `.` are supported in the table names, too; and, in fact, is used in [all "core" tables](https://github.com/exonum/exonum/blob/v0.3/exonum/src/blockchain/schema.rs) and, say, [tables of the configuration service](https://github.com/exonum/exonum-configuration/blob/v0.3/src/lib.rs) (but not in code samples). Should every table name prefixed with `<service_name>.` as a convention?

Reply:
> Should every table name prefixed with <service_name>. as a convention?

Yes, it should. Ideally we want to append a service prefix automatically, but such improvement will not be available soon I guess.